1.2.c

```diff
@@ -0,0 +1,37 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#include <math.h>
+
+int main()
+{
+	printf ("Input a: ");
+	int a;

```

Comment:
```suggestion
	int dividend = 0;
```
1) Инициализируй переменные при объявлении.
2) Говорящие имена.